doc/guide/admin.rst

```diff
@@ -666,22 +666,19 @@ enabled.
 Adding Job Prolog/Epilog Scripts
 ================================
 
-Flux does not currently support a traditional job prolog/epilog
-which runs as root on the nodes assigned to a job before/after job
-execution. Flux does, however, support a job-manager prolog/epilog,
-which is run at the same point on rank 0 as the instance
-owner (typically user ``flux``), instead of user root.
-
-As a temporary solution, a convenience command ``flux perilog-run``
-is provided which can simulate a job prolog and epilog by executing a
-command across the broker ranks assigned to a job from the job-manager
-prolog and epilog.
-
-When using ``flux perilog-run`` to execute job prolog and epilog, the
-job-manager prolog/epilog feature is being used to execute a privileged
-prolog/epilog across the nodes/ranks assigned to a job, via the
-flux-security IMP "run" command support. Therefore, each of these
-components need to be configured, which is explained in the steps below.
+The current best practice for enabling per-job prolog/epilog scripts
+is to use the job-manager prolog/epilog configuration supported by the
+``perilog.so`` jobtap plugin. The perilog plugin optionally initiates a
+prolog action when a job enters the RUN state before any job shells are
+launched, and an epilog action that can be started after all job shells
+complete but before resources are returned to the scheduler.
+
+The perilog plugin now supports directly executing the configured
+prolog/epilog across all ranks of a job, which makes the previous solution
+using ``flux perilog-run`` obsolete.
```

Comment:
Might want to put the bit about the obsolete solution in a Note box.

Reply:
or should it just be removed?